src/sidebar/app/utils/utils.js

```diff
@@ -11,4 +11,45 @@ function formatFooterTime(date) {
   });
 }
 
-export { formatFooterTime };
+/**
+ *
+ * @param {HTMLElement Object} parentElement
+ * @returns {HTMLElement Object}
+ */
+function getFirstNonEmptyElement(parentElement) {
+  // create an Array from parentElement's `children` (HTMLCollection)
+  let parentElementChildrenArray = [];
+  if (parentElement.children.length > 20) {
```

Comment:
@cedricium do you need this check, can you just `.slice(0,20)` no matter what the size is.

Reply:
Check is necessary, otherwise if there are less than 20 elements then `slice` will automatically create an empty array.